tools/bug-report/pkg/bugreport/bugreport.go

```diff
@@ -198,6 +212,43 @@ func runBugReportCommand(_ *cobra.Command, logOpts *log.Options) error {
 	return nil
 }
 
+func isDistroless(ctx context.Context, client kube.CLIClient, clientset *kubernetes.Clientset, brConfig *config.BugReportConfig) (bool, error) {
+	// Get istiod deployment
+	deploy, err := clientset.AppsV1().Deployments(brConfig.IstioNamespace).Get(ctx, "istiod", metav1.GetOptions{})
```

Comment:
Should we put deployment name `istiod` in constant ?

Reply:
Added constant